### PR TITLE
refactor halo2 examples into v022 and v030

### PR DESCRIPTION
**Pull request by katiemckeon**
Updating the structure of our Halo2 examples.  

Since we now support two versions of Halo2 circuitry, I've created two subdirectories that separate our examples by type.  All of the circuits for v022 are preexisting, the v030 circuits are new.

# Reviewer Instructions
- [x] Proofread the new `halo2/README.md`
  - [ ] Proofread both readmes within the v0.2.2 and v0.3.0 subdirectories
- [x] For each sample circuit, upload to Forge and prove.  Ensure there are no errors.
  - [x] Circuit type `Halo2 Axiom v0.2.2`
    - [x] mult-example
    - [x] storage_proof 
  - [x] Circuit type `Halo2 Axiom v0.3.0` 
    - [x] quadratic_circuit
    - [x] float_radius
    - [x] axiom_header_goerli

### CODE REVIEW

**Comment by Jasmine-B23** [comment]
Ran mult-example locally and ran into a few issues. In the `sindri.json` file:
- change `package_name` to `PACKAGE_NAME`
- change 'circuit_path` to `CLASS_NAME`
- add `DEGREE` to the file

I also ran into this error: 
```
replace() argument 2 must be str, not int
```

**Comment by Jasmine-B23** [comment]
Since the above changes have been made, everything LGTM.

File: halo2/axiom-v0.2.2/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    Cargo.toml
+    Sindri.json
+```
+
+The file `Sindri.json` specifies what you have named your package and how to reference the circuit
+```
+{
+    "package_name": "my-circuit",
+    "circuit_path": "circuit::myCircuit"
+}
```
Comment (Jasmine-B23) [inline halo2/axiom-v0.2.2/README.md:35]: Please make sure that the sindri.json updates are reflected here.

**Review by Jasmine-B23 — COMMENTED** [review]
Once the following steps are completed everything will be good to go.

File: halo2/axiom-v0.2.2/README.md (hunk 1 above)
Comment (Jasmine-B23) [reply halo2/axiom-v0.2.2/README.md:35]: same goes for the axiom-v0.3.0/README.md

Comment (katiemckeon) [reply halo2/axiom-v0.2.2/README.md]: Good spotting.  Addressed by https://github.com/Sindri-Labs/forge-sample-data/pull/7/commits/6e84ddce88d910306257780f31ba1c3ffd66af3a

**Review by Jasmine-B23 — APPROVED** [review]
LGTM!

**Pull request merged**